metagraph/tests/algorithms/test_embedding.py

```diff
@@ -3,6 +3,8 @@
 import networkx as nx
 import numpy as np
 import math
+from sklearn.mixture import GaussianMixture
```

Comment:
Now that these are required to run the tests, you should update the docs, specifically the installation section.

> Running tests requires additional libraries to be installed

Add scikit-learn to the list of test dependencies needed.

Reply:
https://github.com/metagraph-dev/metagraph/pull/177/commits/55c9a058fda9f51ca569198109d3f15330d1cb50

This commit updates the docs. It notes that we recommend installing `scikit-learn` for testing purposes and updates what an example run of the test suite looks like.